benchmarks/README.md

```diff
@@ -5,6 +5,27 @@ The two main benchmarking scripts are
   - `result_analyzer.py` to aggregate the benchmark result in CSV form.
 
 
+## Reducing benchmark noise 
+
+It is important to keep the benchmark runs safe from external effects 
+to reduce noise. Run:
+
+```
+# Sets the CPU statically to the highest tuneable frequency. 
+# Prevent energy saving features to kick in.
+sudo cpupower frequency-set --governor performance
+
+# Lock GPU clocks to lower frequency to reduce the chance of extra throttling. Choose
+# FREQ based on your GPU info. For example A100 operates on 765MHz (up to 1410 MHz),
```

Comment:
Could you add the link to help users find the 765MHz and 1215MHz?
Also, regarding `a couple hundrend MHz` below, can it be more quantifiable, for example 90%, 80% etc.?

Reply:
It's hard to tell, personally I set both core clock, and mem clock to 1005MHz for A100. So like 80% of the max clock, but I can imagine it differs across GPUs of different architectures. Sometimes you can even set it to 90% so the device time does not look out of the blue.

Reply:
I added a more general procedure of finding out the frequency. PTAL.